deployment/conf/supported_apps_w_extensions.json

```diff
@@ -145,1007 +145,959 @@
     ]
   },
   "types": {
```

Comment:
Huh, this is the first time I've seen VSCode's diff engine handle a diff and github's completely choke.